statistics-production/embedded-charts.qmd

```diff
@@ -2,15 +2,15 @@
 title: "Embedded visualisations in EES"
 ---
 
-<p class="text-muted">Guidance for creating and embedding R-Shiny visualisations in EES publications</p>
+<p class="text-muted">Guidance for creating and embedding R Shiny visualisations in EES publications</p>
 
 ---
 
 # Introduction
 
 ---
 
-EES now offers the option to add an embedded block, which allows analysts to present custom R created charts within their publications. This necessitates analysts creating an R-Shiny based app on GitHub containing the required chart that can then be hosted on a shiny server (currently using the DfE's account on shinyapps.io).
+EES now offers the option to add an embedded block, which allows analysts to present custom R created charts within their publications. This necessitates analysts creating an R Shiny based app on GitHub containing the required chart that can then be hosted on a shiny server (currently using the DfE's account on shinyapps.io).
```

Comment:
Think this shiny needs a big S also

Reply:
Literally just spotted that level 1 Introduction header above this on line 9 🤣 worth fixing while we're here

Reply:
Big S is fixed though

Reply:
I think every page we haven't recently touched has level 1 headers and the whole rest of the headers on the page also need adjusting to fit... 

I've done this one but I think for anymore we should wait until one big sweep to do the rest as it always requires altering every header on the page, which can take time for bigger pages.